utils/fetchProfile.ts

```diff
@@ -77,6 +85,24 @@ const createProfileObject = async (
   const profileImageId = getHash(profileImage)
   const backgroundImageId = getHash(backgroundImage)
 
+  let receivedAssetAddresses: Address[] = []
+  let issuedAssetAddresses: Address[] = []
+  try {
+    // get received assets array for profile
+    // TODO update this when Algolia provides LSP5 array for the profile
+    const assetAddresses = await erc725.fetchData([
+      'LSP5ReceivedAssets[]',
+      'LSP12IssuedAssets[]',
+    ])
+
+    assertArray(assetAddresses[0].value)
+    assertAddresses(assetAddresses[0].value)
+    receivedAssetAddresses = assetAddresses[0].value
+    assertArray(assetAddresses[1].value)
+    assertAddresses(assetAddresses[1].value)
```

Comment:
can this break? What if the assetAddresses array is empty?

I dont remember how erc725 return the data, but maybe a ? can be added?

```suggestion
    assertArray(assetAddresses[0]?.value)
    assertAddresses(assetAddresses[0]?.value)
    receivedAssetAddresses = assetAddresses[0]?.value
    assertArray(assetAddresses[1]?.value)
    assertAddresses(assetAddresses[1]?.value)
```

Reply:
Not sure either, but `?` makes sense just in case 😉